virttest/utils_libvirt/libvirt_network.py

```diff
@@ -346,10 +346,29 @@ def change_tcp_config(expected_dict):
 
 def check_listen_address(params):
```

Comment:
Maybe we rename it to like `check_sockets_statistics(check_patterns)`

Reply:
Updated, thanks.